### DISCUSSION TITLE
idle shutdown

### DISCUSSION THREAD

**Discussion by ghost** [coder/code-server#6169] [Q&A]
Does the code-server have the function of idle shutdown
Similar to this instruct
--enable-remote-auto-shutdown
--remote-auto-shutdown-without-delay

**Comment by code-asher** [comment]
This does not exist yet.  Relevant issue: https://github.com/coder/code-server/issues/5008



You could watch the heartbeat file (https://github.com/coder/code-server/pull/1115) and use it to kill code-server when it stops updating although this is not real idle, just indicates that there are no active connections.
